api/v1/bmc.proto

```diff
@@ -82,3 +70,21 @@ message UpdateUserRequest {
 message UpdateUserResponse {
     string task_id = 1;
 }
+
+enum UserRole {
+    USER_ROLE_UNSPECIFIED = 0;
+    USER_ROLE_ADMIN = 1;
+    USER_ROLE_USER = 2;
+}
```

Comment:
Will this change the value returned when doing `Role.String()`? Previously that would return just `admin` or `user`. I'd suspect it would now return `USER_ROLE_ADMIN` instead.

Reply:
ah, that's fair. I did not check. I will do that now.

Reply:
@mikemrm pushed a change for this.